Detections/AWSCloudTrail/AWS_ChangeToRDSDatabase.yaml

```diff
@@ -19,12 +21,10 @@ relevantTechniques:
   - T1098
 query: |
 
-  let timeframe = 1d;
+  let timeframe = 1h;
+  let EventNameList = dynamic(["AuthorizeDBSecurityGroupIngress","CreateDBSecurityGroup","DeleteDBSecurityGroup","RevokeDBSecurityGroupIngress"]);
   AWSCloudTrail
-  | where TimeGenerated >= ago(timeframe)
-  | where EventName == "AuthorizeDBSecurityGroupIngress"
-      or EventName == "CreateDBSecurityGroup"
-      or EventName == "DeleteDBSecurityGroup"
-      or EventName == "RevokeDBSecurityGroupIngress"
+  | where TimeGenerated > ago(timeframe)
+  | where EventName in (EventNameList)
   | project TimeGenerated, EventName, EventTypeName, UserIdentityAccountId, UserIdentityPrincipalid, UserAgent, UserIdentityUserName, SessionMfaAuthenticated, SourceIpAddress, AWSRegion, EventSource, AdditionalEventData, ResponseElements
```

Comment:
Can we aggregate on time here to reduce the rowset that is returned ie. StartTimeUtc = min(TimeGenerated)?  Also, any other aggregation points that will also help reduce the rowset?